data/hai/unfettered side missions.txt

```diff
@@ -203,29 +139,55 @@ mission "Hai Legacy 1"
+			`	"This ship is our... guarantee that nothing goes wrong, we are still not sure if we can trust you, after all," says the second one.`
+			`	The first one speaks again, after taking you to a nearby cave to show you another ship. "We would never sell you that junk. This ship, however, is a mighty relic of the past, intended for war from the times our Empire used to control the stars: an Anomalocaris!`
+			scene "ship/hai anomalocaris"
+			`	The second one now takes a step towards you to speak, almost interrupting the first, "We have a proposition for you: should you agree to bring much needed supplies to our brethren, the Unfettered Hai, we will sell you this ship for the modest price of 10 million credits, including 2 very powerful Korath Detainers!" The first one seems surprised by this and tries to intervene, but is stopped by a sign of the hand. The second resumes, "You may have heard of those in legends as weapons able to almost disable any engines to give you the speed advantage. Worry not, you'll be able to recuperate the cost of this ship with ease by bringing medical supplies to the Unfettered, or by mining in the nearby systems. This cargo is needed on short notice, on <destination>, and we lack the time and personnel to bring them.`
 			`	"Of course, the ship is aged, and you should not expect the best price in a shipyard. Nevertheless, it is a fine vessel, capable of fighting, freighting, and mining. So, what do you say, <last>? This is the opportunity of a lifetime; they simply don't make ships like this anymore."`
+			label choice
 			choice
-				`	"I'll take it!"`
+				`	"I'll take that... Anomalocaris?"`
 					goto take
+				`	"If you lack crew, who was piloting that Ladybug?"`
+					to display
+						not "unfettered: pilot dialog"
+					goto pilot
+				`	"Why was this ship submerged?"`
+					to display
+						not "unfettered: ocean dialog"
+					goto ocean
 				`	"No thanks. I'm not looking to add another ship to my fleet."`
 			`	"A shame. We will probably not meet again. Farewell," one says. They signal the ship to submerge into the ocean again and walk off along the beach.`
 				goto trainback
+			label pilot
+			action
+				set "unfettered: pilot dialog"
+			`	"You mean what is-" says the first one, stopped by the second, "The less you know, the better for both of us, Captain."`
+				goto choice
+			label ocean
+			action
+				set "unfettered: ocean dialog"
+			`	"We found it there," the first one begins, "It is hard to find a drop of rain in a storm, but the drop of rain may see all, and we were that drop. Worry not, we have restored its cuirass." the other one adds.`
```

Comment:
```suggestion
			`	"We found it there," the first one answers. "It is hard to find a drop of rain in a storm, but the drop of rain may see all, and we were that drop. Worry not, we have restored its cuirass."`
```
What does `cuirass` mean in this context? Seems like an unusual word choice. I'd suggest `chassis`, `frame` or something like those.

Reply:
good point, it just means the hull of the ship

Reply:
```suggestion
			`	"We found it there," the first one answers, "It is hard to find a drop of rain in a storm, but the drop of rain may see all, and we were that drop. Worry not, we have restored its hull." the other one adds.`
```

Reply:
Sure, we can keep the alternating discussion between the first and second. Here's a writing thing tov be aware of: if the speaker (person talking) changes, you have to start a new paragraph. So it would look something like this:
```suggestion
			`	"We found it there," the first one answers.`
			`	"It is hard to find a drop of rain in a storm, but the drop of rain may see all, and we were that drop. Worry not, we have restored its hull," the other one adds.`
```

Reply:
I didnt do it initially bc does the world know of it?
but yeah we know of it that's true and we should use it